### PR TITLE
Project view have projects navigation

### PR DESCRIPTION
**Pull request by mkasztelnik**
In the scope of this PR we will rearrange all projects views.

### CODE REVIEW

**Comment by roksanaer** [comment]
quick remark. i did early tests. after improved redirecton from ordering (to the project/project item) in this PR we lost a possibility to add another service (see screens)
obraz
obraz

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -90,6 +93,10 @@ def destroy
 
   private
 
+    def projectless?
```
Comment (martaswiatkowska) [inline app/controllers/projects_controller.rb]: Where do you use this?

**Comment by mkasztelnik** [comment]
@agpul fixed, now after project create user is redirected to the new project instead of the first project.

File: app/controllers/projects_controller.rb (hunk 1 above)
Comment (mkasztelnik) [reply app/controllers/projects_controller.rb]: Good catch, I already removed it.

**Pull request merged**